cli/cli/command_framework/highlevel/file_system_path_arg/file_system_path_arg.go

```diff
@@ -11,65 +11,92 @@ import (
 
 const (
 	isNotGreedyArg = false
-	defaultValue   = ""
+	ContinueWithDefaultValidation = true
+	DoNotContinueWithDefaultValidation = false
 )
 
-// This function type can be used to create custom argument value conditions where the validation will be omitted
-// To omit the argument value the function has to return true when the condition is accomplished
-type fileSystemArgumentValidationExceptionFunc func(argumentValue string) (isValid bool)
-var NoValidationExceptionFunc fileSystemArgumentValidationExceptionFunc = nil
+// The File System Path Arg comes with a default validation function (see below getValidationFunc)
+// A custom validation function can be used to customize the validation.  This validation
```

Comment:
```suggestion
// A custom validation function can be used to customize the validation. This validation
```
removed double space before `This`